utils/rpms/packaging/rpm_chrootbuild

```diff
@@ -114,6 +118,10 @@ if [ -n "$DISTRO_VERSION" ]; then
     releasever_opt=("--config-opts=releasever=$DISTRO_VERSION")
 fi
 # shellcheck disable=SC2086
-eval mock --configdir "$mock_config_dir" -r "${CHROOT_NAME}"           \
-     ${repo_dels[*]} ${repo_adds[*]} --disablerepo=\*-debug*           \
-     "${releasever_opt[@]}" $MOCK_OPTIONS $RPM_BUILD_OPTIONS "$TARGET"
+if ! eval mock --configdir "$mock_config_dir" -r "${CHROOT_NAME}"           \
+          ${repo_dels[*]} ${repo_adds[*]} --disablerepo=\*-debug*           \
```

Comment:
(lint) Use "${array[@]}" (with quotes) to prevent whitespace problems. [SC2048]
(lint) Use "${array[@]}" (with quotes) to prevent whitespace problems. [SC2048]